<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>LearningWindow</class>
 <widget class="QDialog" name="LearningWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>628</width>
    <height>287</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Обучение</string>
  </property>
  <property name="windowIcon">
   <iconset resource="source.qrc">
    <normaloff>:/icons/favicon.png</normaloff>:/icons/favicon.png</iconset>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout_3">
   <item>
    <layout class="QHBoxLayout" name="horizontalLayout_6">
     <item>
      <layout class="QVBoxLayout" name="verticalLayout">
       <item>
        <layout class="QHBoxLayout" name="horizontalLayout">
         <item>
          <widget class="QLabel" name="trainSizeLbl">
           <property name="text">
            <string>Размер тестовой выборки</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QSlider" name="trainSizeSldr">
           <property name="maximum">
            <number>100</number>
           </property>
           <property name="orientation">
            <enum>Qt::Horizontal</enum>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QDoubleSpinBox" name="trainSizeSB">
           <property name="buttonSymbols">
            <enum>QAbstractSpinBox::NoButtons</enum>
           </property>
          </widget>
         </item>
        </layout>
       </item>
       <item>
        <layout class="QHBoxLayout" name="horizontalLayout_2">
         <item>
          <widget class="QLabel" name="numEpochsLbl">
           <property name="text">
            <string>Количество эпох</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QSpinBox" name="numEpochsSB"/>
         </item>
        </layout>
       </item>
       <item>
        <layout class="QHBoxLayout" name="horizontalLayout_3">
         <item>
          <widget class="QLabel" name="batchSizeLbl">
           <property name="text">
            <string>Размер батча</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QSpinBox" name="batchSizeSB"/>
         </item>
        </layout>
       </item>
       <item>
        <layout class="QHBoxLayout" name="horizontalLayout_4">
         <item>
          <widget class="QLabel" name="learningRateLbl">
           <property name="text">
            <string>Скорость обучения</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QDoubleSpinBox" name="learningRateSB"/>
         </item>
        </layout>
       </item>
       <item>
        <widget class="QGroupBox" name="convGB">
         <property name="title">
          <string>Свертка</string>
         </property>
         <layout class="QFormLayout" name="formLayout">
          <item row="0" column="0">
           <widget class="QLabel" name="kernelSizeLbl">
            <property name="text">
             <string>Размер ядра</string>
            </property>
           </widget>
          </item>
          <item row="0" column="1">
           <widget class="QSpinBox" name="kernelSizeSB"/>
          </item>
          <item row="1" column="0">
           <widget class="QLabel" name="stepLbl">
            <property name="text">
             <string>Шаг</string>
            </property>
           </widget>
          </item>
          <item row="1" column="1">
           <widget class="QSpinBox" name="stepSB"/>
          </item>
          <item row="2" column="0">
           <widget class="QLabel" name="paddingLbl">
            <property name="text">
             <string>Отступ</string>
            </property>
           </widget>
          </item>
          <item row="2" column="1">
           <widget class="QSpinBox" name="paddingSB"/>
          </item>
         </layout>
        </widget>
       </item>
      </layout>
     </item>
     <item>
      <widget class="Line" name="line">
       <property name="orientation">
        <enum>Qt::Vertical</enum>
       </property>
      </widget>
     </item>
     <item>
      <layout class="QVBoxLayout" name="verticalLayout_2">
       <item>
        <widget class="QLabel" name="accuracyHeaderLbl">
         <property name="text">
          <string>Точность на тестовой выборке</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QLabel" name="accuracyLbl">
         <property name="font">
          <font>
           <pointsize>24</pointsize>
          </font>
         </property>
         <property name="text">
          <string/>
         </property>
        </widget>
       </item>
      </layout>
     </item>
    </layout>
   </item>
   <item>
    <layout class="QHBoxLayout" name="horizontalLayout_5">
     <item>
      <widget class="QPushButton" name="pushButton">
       <property name="text">
        <string>Обучить</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QProgressBar" name="progressBar">
       <property name="value">
        <number>0</number>
       </property>
      </widget>
     </item>
    </layout>
   </item>
  </layout>
 </widget>
 <resources>
  <include location="source.qrc"/>
 </resources>
 <connections/>
</ui>
